src/hubverse_infrastructure/hubs/hubs.yaml

```diff
@@ -14,4 +14,6 @@ hubs:
 - hub: temp-flusight-archive-hub
```

Comment:
Will submit a follow-up PR to remove the temporary version of the FluSight archive.